ssl/ssl_lib.c

```diff
@@ -675,6 +679,11 @@ int SSL_is_dtls(const SSL *s)
     return SSL_IS_DTLS(s) ? 1 : 0;
 }
 
+int SSL_is_tls13_plus(const SSL *s)
```

Comment:
It was an attempt at future proofing, so that it isn't confusing when we have TLSv1.4. The idea was that it should return 1 if you have a TLSv1.3 or later protocol. I'm not especially happy with the name, except to say that I think `SSL_is_tls13()` is worse.